iree/turbine/kernel/wave/perf/benchmark.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3
```

Comment:
Do we need to tell the python path explicitly?

Reply:
I can remove this.